cime_config/testdefs/testlist_clm.xml

```diff
@@ -17,15 +17,6 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
-  <test name="SMS_D_Ln6" grid="f09_g17" compset="I1850Clm50BgcNoAnthro" testmods="clm/Meier2022_surf_rough">
```

Comment:
Removing the Meier testmods made this test redundant as a clm50 test, so I removed it. 
(I did first try changing to a clm51 test and got `Invalid compset name, I1850Clm51BgcNoAnthro`.)